sympy/stats/stochastic_process_types.py

```diff
@@ -413,7 +414,7 @@ def _check_trans_probs(self, trans_probs, row_sum=1):
         if not isinstance(trans_probs, MatrixSymbol):
             rows = trans_probs.tolist()
             for row in rows:
-                if (sum(row) - row_sum) != 0:
+                if abs(sum(row) - row_sum) > 1e-5:
```

Comment:
L419 suggests to use `Rational`. Probably reviews from more experienced reviewer are needed to decide upon this. @oscarbenjamin @asmeurer @Upabjojr

Reply:
Error pops up despite using `Rational`. This is on Sympy Live
``` python
>>> from sympy import *
>>> from sympy.stats import *
>>> T = Matrix([[Rational(0.5), Rational(0.4), Rational(0.1)], [Rational(0.5), Rational(0.4), Rational(0.1)], [Rational(0.5), Rational(0.4), Rational(0.1)]])
>>> Y = DiscreteMarkovChain("Y", [0, 1, 2], T)
>>> P(Eq(Y[1], 0), Eq(Y[0], 0))
  File "/base/data/home/apps/s~sympy-live-hrd/66.426491309333028408/sympy/sympy/stats/stochastic_process_types.py", line 280, in _check_trans_probs
    "If you are using Float or floats then please use Rational."%(row_sum))
ValueError: Values in a row must sum to 1. If you are using Float or floats then please use Rational.
```

Reply:
```python
In [1]: Rational(1, 4)
Out[1]: 1/4

In [2]: Rational(0.4)
Out[2]: 
3602879701896397
────────────────
9007199254740992
```

Reply:
Definitely a bad edit.

Reply:
You shouldn't be using floating-point numbers in SymPy, if possible.

Reply:
My bad, I'll change it back to what it was and change the tests too.